---
title: Publication
permalink: /publication/
---

For those interested in numbers, see Brian's [Google Scholar citations profile](https://scholar.google.com/citations?user=0EPfe4UAAAAJ).

If you cannot access one of our papers, let us know and we'll try to help. 

<hr>

### 2025

Gan YJ, Hazel JM, **Searle BC**, Shafaat HS. Selective isotope labeling probes
the chemical capacity and reaction mechanism of a heterobimetallic Mn/Fe
protein. J Inorg Biochem. 2025 Apr 23;270:112933. doi:
10.1016/j.jinorgbio.2025.112933. Epub ahead of print. PMID: 40378440.

**Shannon AE**, Teodorescu RN, Song NJ, Heil LR, Jacob CC, Remes PM, Li Z,
Rubinstein MP, **Searle BC**. Rapid assay development for low input targeted
proteomics using a versatile linear ion trap. Nat Commun. 2025 Apr
23;16(1):3794. doi: 10.1038/s41467-025-58757-8. PMID: 40263265; PMCID:
PMC12015518.

Rajczewski AT, Blakeley-Ruiz JA, Meyer A, Vintila S, McIlvin MR, Van Den
Bossche T, **Searle BC**, Griffin TJ, Saito MA, Kleiner M, Jagtap PD. Data-
Independent Acquisition Mass Spectrometry as a Tool for Metaproteomics:
Interlaboratory Comparison Using a Model Microbiome. Proteomics. 2025
May;25(9-10):e202400187. doi: 10.1002/pmic.202400187. Epub 2025 Apr 10. PMID:
40211604.

**Searle BC**, Chazarin B, Collins BC, Kundu DJ, Huang S, Lin Q, Liu Y, Low TY,
Saba J, Guo T, Palmisano G, Fert-Bober J. Advancing Scientific Communication in
Proteomics. J Proteome Res. 2025 Feb 7;24(2):381-382. doi:
10.1021/acs.jproteome.4c01098. PMID: 39916559.

### 2024

**Searle BC**. Characterizing protein-protein interactions with thermal proteome
profiling. Curr Opin Struct Biol. 2024 Dec;89:102946. doi:
10.1016/j.sbi.2024.102946. Epub 2024 Oct 30. PMID: 39481280; PMCID: PMC11602378.

**Ghosh G**, **Shannon AE**, **Searle BC**. Data acquisition approaches for single cell
proteomics. Proteomics. 2025 Jan;25(1-2):e2400022. doi: 10.1002/pmic.202400022.
Epub 2024 Aug 1. PMID: 39088833; PMCID: PMC11735665.

Akshintala VS, **Moore MG**, Cruz-Monserrate Z, Nathan JD, **Searle BC**, Abu-El-
Haija M. Urine Proteomics Profiling Identifies Novel Acute Pancreatitis
Diagnostic Biomarkers in a Pediatric Population. Gastroenterology. 2024
Oct;167(5):1019-1021.e2. doi: 10.1053/j.gastro.2024.05.015. Epub 2024 May 24.
PMID: 38797238; PMCID: PMC11416306.

**Searle BC**. Nanopore Protein Sequencing Achieves Significant New Milestones.
Clin Chem. 2024 Aug 1;70(8):1006-1008. doi: 10.1093/clinchem/hvae041. PMID:
38593193.

**Shannon AE**, Boos CE, **Searle BC**, Hummon AB. Gas-Phase Fractionation Data-
Independent Acquisition Analysis of 3D Cocultured Spheroid Tumor Model Reveals
Altered Translational Processes and Signaling Using Proteomics. J Proteome Res.
2024 Aug 2;23(8):3188-3199. doi: 10.1021/acs.jproteome.3c00786. Epub 2024 Feb 27. PMID: 38412258; PMCID: PMC11296903.

**Joyce AW**, **Searle BC**. Computational approaches to identify sites of
phosphorylation. Proteomics. 2024 Apr;24(8):e2300088. doi:
10.1002/pmic.202300088. Epub 2023 Dec 24. PMID: 37897210.

### 2023

Heil LR, Damoc E, Arrey TN, Pashkova A, Denisov E, Petzoldt J, Peterson AC,
Hsu C, **Searle BC**, Shulman N, Riffle M, Connolly B, MacLean BX, Remes PM, Senko
MW, Stewart HI, Hock C, Makarov AA, Hermanson D, Zabrouskov V, Wu CC, MacCoss
MJ. Evaluating the Performance of the Astral Mass Analyzer for Quantitative
Proteomics Using Data-Independent Acquisition. J Proteome Res. 2023 Oct
6;22(10):3290-3300. doi: 10.1021/acs.jproteome.3c00357. Epub 2023 Sep 8. PMID:
37683181; PMCID: PMC10563156.

**Searle BC**, Chien A, Koller A, Hawke D, Herren AW, Kim Kim J, Lee KA, Leib
RD, Nelson AJ, Patel P, Ren JM, Stemmer PM, Zhu Y, Neely BA, Patel B. A
Multipathway Phosphopeptide Standard for Rapid Phosphoproteomics Assay
Development. Mol Cell Proteomics. 2023 Oct;22(10):100639. doi:
10.1016/j.mcpro.2023.100639. Epub 2023 Aug 30. PMID: 37657519; PMCID:
PMC10561125.

Kirkpatrick J, Stemmer PM, **Searle BC**, Herring LE, Martin L, Midha MK,
Phinney BS, Shan B, Palmblad M, Wang Y, Jagtap PD, Neely BA. 2019 Association of
Biomolecular Resource Facilities Multi-Laboratory Data-Independent Acquisition
Proteomics Study. J Biomol Tech. 2023 Jun 2;34(2):3fc1f5fe.9b78d780. doi:
10.7171/3fc1f5fe.9b78d780. PMID: 37435391; PMCID: PMC10332336.

Allen C, Meinl R, Paez JS, **Searle BC**, Just S, Pino LK, Fondrie WE. nf-
encyclopedia: A Cloud-Ready Pipeline for Chromatogram Library Data-Independent
Acquisition Proteomics Workflows. J Proteome Res. 2023 Aug 4;22(8):2743-2749.
doi: 10.1021/acs.jproteome.2c00613. Epub 2023 Jul 7. PMID: 37417926.

**Phlairaharn T**, Ye Z, Krismer E, Pedersen AK, Pietzner M, Olsen JV, Schoof
EM, **Searle BC**. Optimizing Linear Ion-Trap Data-Independent Acquisition toward
Single-Cell Proteomics. Anal Chem. 2023 Jul 4;95(26):9881-9891. doi:
10.1021/acs.analchem.3c00842. Epub 2023 Jun 20. PMID: 37338819.

Merrihew GE, Park J, Plubell D, **Searle BC**, Keene CD, Larson EB, Bateman R,
Perrin RJ, Chhatwal JP, Farlow MR, McLean CA, Ghetti B, Newell KL, Frosch MP,
Montine TJ, MacCoss MJ. A peptide-centric quantitative proteomics dataset for
the phenotypic assessment of Alzheimer's disease. Sci Data. 2023 Apr
14;10(1):206. doi: 10.1038/s41597-023-02057-7. PMID: 37059743; PMCID:
PMC10104800.

**Searle BC**, **Shannon AE**, **Wilburn DB**. Scribe: Next Generation Library Searching
for DDA Experiments. J Proteome Res. 2023 Feb 3;22(2):482-490. doi:
10.1021/acs.jproteome.2c00672. Epub 2023 Jan 25. PMID: 36695531.

Li W, Pergande MR, Crutchfield CA, **Searle BC**, Backlund PS, Picache JA,
Burkert K, Yanjanin-Farhat NM, Blank PS, Toth CL, Wassif CA, Porter FD, Cologna
SM. A differential proteomics study of cerebrospinal fluid from individuals with
Niemann-Pick disease, Type C1. Proteomics. 2023 Jun;23(11):e2200378. doi:
10.1002/pmic.202200378. Epub 2023 Jan 26. PMID: 36638187; PMCID: PMC10918788.

Martinez TF, Lyons-Abbott S, Bookout AL, De Souza EV, Donaldson C, Vaughan
JM, Lau C, Abramov A, Baquero AF, Baquero K, Friedrich D, Huard J, Davis R, Kim
B, Koch T, Mercer AJ, Misquith A, Murray SA, Perry S, Pino LK, Sanford C, Simon
A, Zhang Y, Zipp G, Bizarro CV, Shokhirev MN, Whittle AJ, **Searle BC**, MacCoss MJ,
Saghatelian A, Barnes CA. Profiling mouse brown and white adipocytes to identify
metabolically relevant small ORFs and functional microproteins. Cell Metab. 2023
Jan 3;35(1):166-183.e11. doi: 10.1016/j.cmet.2022.12.004. PMID: 36599300; PMCID:
PMC9889109.

### 2022

11: Phlairaharn T, Grégoire S, Woltereck LR, Petrosius V, Furtwängler B, **Searle
BC**, Schoof EM. _High Sensitivity Limited Material Proteomics Empowered by Data-
Independent Acquisition on Linear Ion Traps._ J Proteome Res. 2022 Nov
4;21(11):2815-2826. doi: 10.1021/acs.jproteome.2c00376. Epub 2022 Oct 26. PMID:
36287219.

12: Richards AL, Chen KH, **Wilburn DB**, Stevenson E, Polacco BJ, **Searle BC**, Swaney
DL. _Data-Independent Acquisition Protease-Multiplexing Enables Increased
Proteome Sequence Coverage Across Multiple Fragmentation Modes._ J Proteome Res.
2022 Apr 1;21(4):1124-1136. doi: 10.1021/acs.jproteome.1c00960. Epub 2022 Mar 2.
PMID: 35234472; PMCID: PMC9035370.

13: **Wilburn DB**, Kunkel CL, Feldhoff RC, Feldhoff PW, **Searle BC**. _Recurrent Co-
Option and Recombination of Cytokine and Three Finger Proteins in Multiple
Reproductive Tissues Throughout Salamander Evolution._ Front Cell Dev Biol. 2022
Feb 23;10:828947. doi: 10.3389/fcell.2022.828947. PMID: 35281090; PMCID:
PMC8904931.

### 2021

McEllin B, **Searle BC**, DePledge L, Sun G, Cobbs C, Karimi M. _Detection of
Human Papillomavirus Integration in Brain Metastases from Oropharyngeal Tumors
by Targeted Sequencing._ Viruses. 2021 Aug 3;13(8):1536. doi: 10.3390/v13081536.
PMID: 34452401; PMCID: PMC8402651.

**Wilburn DB**, Richards AL, Swaney DL, **Searle BC**. _CIDer: A Statistical Framework
for Interpreting Differences in CID and HCD Fragmentation._ J Proteome Res. 2021
Apr 2;20(4):1951-1965. doi: 10.1021/acs.jproteome.0c00964. Epub 2021 Mar 17.
PMID: 33729787.

### 2020

Robinson AE, Binek A, Venkatraman V, **Searle BC**, Holewinski RJ, Rosenberger G,
Parker SJ, Basisty N, Xie X, Lund PJ, Saxena G, Mato JM, Garcia BA, Schilling B,
Lu SC, Van Eyk JE. _Lysine and Arginine Protein Post-translational Modifications
by Enhanced DIA Libraries: Quantification in Murine Liver Disease._ J Proteome
Res. 2020 Oct 2;19(10):4163-4178. doi: 10.1021/acs.jproteome.0c00685. Epub 2020
Sep 23. PMID: 32966080.

Pino LK, Just SC, MacCoss MJ, **Searle BC**. _Acquiring and Analyzing Data
Independent Acquisition Proteomics Experiments without Spectrum Libraries._ Mol
Cell Proteomics. 2020 Jul;19(7):1088-1103. doi: 10.1074/mcp.P119.001913. Epub
2020 Apr 20. PMID: 32312845; PMCID: PMC7338082.

Mische SM, Fisher NC, Meyn SM, Sol-Church K, Hegstad-Davies RL, Weis-Garcia
F, Adams M, Ashton JM, Delventhal KM, Dragon JA, Holmes L, Jagtap P, Kubow KE,
Mason CE, Palmblad M, **Searle BC**, Turck CW, Knudtson KL. _A Review of the
Scientific Rigor, Reproducibility, and Transparency Studies Conducted by the
ABRF Research Groups._ J Biomol Tech. 2020 Apr;31(1):11-26. doi:
10.7171/jbt.20-3101-003. PMID: 31969795; PMCID: PMC6959150.

**Searle BC**, Swearingen KE, Barnes CA, Schmidt T, Gessulat S, Küster B, Wilhelm
M. _Generating high quality libraries for DIA MS with empirically corrected
peptide predictions._ Nat Commun. 2020 Mar 25;11(1):1548. doi:
10.1038/s41467-020-15346-1. PMID: 32214105; PMCID: PMC7096433.

Federation AJ, Nandakumar V, **Searle BC**, Stergachis A, Wang H, Pino LK,
Merrihew G, Ting YS, Howard N, Kutyavin T, MacCoss MJ, Stamatoyannopoulos JA.
_Highly Parallel Quantification and Compartment Localization of Transcription
Factors and Nuclear Proteins._ Cell Rep. 2020 Feb 25;30(8):2463-2471.e5. doi:
10.1016/j.celrep.2020.01.096. PMID: 32101728.

Pino LK, **Searle BC**, Yang HY, Hoofnagle AN, Noble WS, MacCoss MJ. _Matrix-
Matched Calibration Curves for Assessing Analytical Figures of Merit in
Quantitative Proteomics._ J Proteome Res. 2020 Mar 6;19(3):1147-1153. doi:
10.1021/acs.jproteome.9b00666. Epub 2020 Feb 24. PMID: 32037841; PMCID:
PMC7175947.

Johnson RS, **Searle BC**, Nunn BL, Gilmore JM, Phillips M, Amemiya CT, Heck M,
MacCoss MJ. _Assessing Protein Sequence Database Suitability Using De Novo
Sequencing. Mol Cell Proteomics._ 2020 Jan;19(1):198-208. doi:
10.1074/mcp.TIR119.001752. Epub 2019 Nov 15. PMID: 31732549; PMCID: PMC6944239.

### 2019

**Searle BC**, Lawrence RT, MacCoss MJ, Villén J. _Thesaurus: quantifying
phosphopeptide positional isomers._ Nat Methods. 2019 Aug;16(8):703-706. doi:
10.1038/s41592-019-0498-4. Epub 2019 Jul 29. PMID: 31363206; PMCID: PMC7012383.

**Searle BC**, Yergey AL. _An efficient solution for resolving iTRAQ and TMT
channel cross-talk._ J Mass Spectrom. 2020 Aug;55(8):e4354. doi:
10.1002/jms.4354. Epub 2019 Apr 29. PMID: 30882954.

Kim YJ, Sweet SMM, Egertson JD, Sedgewick AJ, Woo S, Liao WL, Merrihew GE,
**Searle BC**, Vaske C, Heaton R, MacCoss MJ, Hembrough T. _Data-Independent
Acquisition Mass Spectrometry To Quantify Protein Levels in FFPE Tumor Biopsies
for Molecular Diagnostics._ J Proteome Res. 2019 Jan 4;18(1):426-435. doi:
10.1021/acs.jproteome.8b00699. Epub 2018 Dec 12. PMID: 30481034; PMCID:
PMC6465117.

### 2018

**Searle BC**, Pino LK, Egertson JD, Ting YS, Lawrence RT, MacLean BX, Villén J,
MacCoss MJ. _Chromatogram libraries improve peptide detection and quantification
by data independent acquisition mass spectrometry._ Nat Commun. 2018 Dec
3;9(1):5128. doi: 10.1038/s41467-018-07454-w. PMID: 30510204; PMCID: PMC6277451.

Pino LK, **Searle BC**, Huang EL, Noble WS, Hoofnagle AN, MacCoss MJ.
_Calibration Using a Single-Point External Reference Material Harmonizes
Quantitative Mass Spectrometry Proteomics Data between Platforms and
Laboratories._ Anal Chem. 2018 Nov 6;90(21):13112-13117. doi:
10.1021/acs.analchem.8b04581. Epub 2018 Oct 23. PMID: 30350613; PMCID:
PMC6854904.

Seitzer PM, **Searle BC**. _Incorporating In-Source Fragment Information Improves
Metabolite Identification Accuracy in Untargeted LC-MS Data Sets._ J Proteome
Res. 2019 Feb 1;18(2):791-796. doi: 10.1021/acs.jproteome.8b00601. Epub 2018 Oct
18. PMID: 30295490.

### 2017

Ting YS, Egertson JD, Bollinger JG, **Searle BC**, Payne SH, Noble WS, MacCoss
MJ. _PECAN: library-free peptide detection for data-independent acquisition
tandem mass spectrometry data._ Nat Methods. 2017 Sep;14(9):903-908. doi:
10.1038/nmeth.4390. Epub 2017 Aug 7. PMID: 28783153; PMCID: PMC5578911.

Pino LK, **Searle BC**, Bollinger JG, Nunn B, MacLean B, MacCoss MJ. _The Skyline
ecosystem: Informatics for quantitative mass spectrometry proteomics._ Mass
Spectrom Rev. 2020 May;39(3):229-244. doi: 10.1002/mas.21540. Epub 2017 Jul 9.
PMID: 28691345; PMCID: PMC5799042.

### 2016

**Searle BC**, Gittelman RM, Manor O, Akey JM. _Detecting Sources of
Transcriptional Heterogeneity in Large-Scale RNA-Seq Data Sets._ Genetics. 2016
Dec;204(4):1391-1396. doi: 10.1534/genetics.116.193714. Epub 2016 Oct 11. PMID:
27729424; PMCID: PMC5161273.

Epstein JA, Blank PS, **Searle BC**, Catlin AD, Cologna SM, Olson MT, Backlund
PS, Coorssen JR, Yergey AL. _ProteinProcessor: A probabilistic analysis using
mass accuracy and the MS spectrum._ Proteomics. 2016 Sep;16(18):2480-90. doi:
10.1002/pmic.201600137. PMID: 27546229; PMCID: PMC5176252.

Lawrence RT, **Searle BC**, Llovet A, Villén J. _Plug-and-play analysis of the
human phosphoproteome by targeted high-resolution mass spectrometry._ Nat
Methods. 2016 May;13(5):431-4. doi: 10.1038/nmeth.3811. Epub 2016 Mar 28. PMID:
27018578; PMCID: PMC5915315.

### 2015

Cologna SM, Crutchfield CA, **Searle BC**, Blank PS, Toth CL, Ely AM, Picache
JA, Backlund PS, Wassif CA, Porter FD, Yergey AL. _An Efficient Approach to
Evaluate Reporter Ion Behavior from MALDI-MS/MS Data for Quantification Studies
Using Isobaric Tags._ J Proteome Res. 2015 Oct 2;14(10):4169-78. doi:
10.1021/acs.jproteome.5b00254. Epub 2015 Sep 3. PMID: 26288259; PMCID:
PMC5571863.

**Searle BC**, Egertson JD, Bollinger JG, Stergachis AB, MacCoss MJ. _Using Data
Independent Acquisition (DIA) to Model High-responding Peptides for Targeted
Proteomics Experiments._ Mol Cell Proteomics. 2015 Sep;14(9):2331-40. doi:
10.1074/mcp.M115.051300. Epub 2015 Jun 22. PMID: 26100116; PMCID: PMC4563719.

### 2014

Seymour SL, Farrah T, Binz PA, Chalkley RJ, Cottrell JS, **Searle BC**, Tabb DL,
Vizcaíno JA, Prieto G, Uszkoreit J, Eisenacher M, Martínez-Bartolomé S, Ghali F,
Jones AR. _A standardized framing for reporting protein identifications in
mzIdentML 1.2._ Proteomics. 2014 Nov;14(21-22):2389-99. doi:
10.1002/pmic.201400080. Epub 2014 Sep 23. PMID: 25092112; PMCID: PMC4384534.

### 2013

Ivanov AR, Colangelo CM, Dufresne CP, Friedman DB, Lilley KS, Mechtler K,
Phinney BS, Rose KL, Rudnick PA, **Searle BC**, Shaffer SA, Weintraub ST.
_Interlaboratory studies and initiatives developing standards for proteomics._
Proteomics. 2013 Mar;13(6):904-9. doi: 10.1002/pmic.201200532. Epub 2013 Feb 19.
PMID: 23319436; PMCID: PMC3863631.

### 2012

Jones AR, Eisenacher M, Mayer G, Kohlbacher O, Siepen J, Hubbard SJ, Selley
JN, **Searle BC**, Shofstahl J, Seymour SL, Julian R, Binz PA, Deutsch EW, Hermjakob
H, Reisinger F, Griss J, Vizcaíno JA, Chambers M, Pizarro A, Creasy D. _The
mzIdentML data standard for mass spectrometry-based proteomics results._ Mol Cell
Proteomics. 2012 Jul;11(7):M111.014381. doi: 10.1074/mcp.M111.014381. Epub 2012
Feb 27. PMID: 22375074; PMCID: PMC3394945.

### 2011

Eng JK, **Searle BC**, Clauser KR, Tabb DL. _A face in the crowd: recognizing
peptides through database search._ Mol Cell Proteomics. 2011
Nov;10(11):R111.009522. doi: 10.1074/mcp.R111.009522. Epub 2011 Aug 29. PMID:
21876205; PMCID: PMC3226415.

### 2010 and earlier

**Searle BC**. _Scaffold: a bioinformatic tool for validating MS/MS-based
proteomic studies._ Proteomics. 2010 Mar;10(6):1265-9. doi:
10.1002/pmic.200900437. PMID: 20077414.

**Searle BC**, Turner M, Nesvizhskii AI. _Improving sensitivity by
probabilistically combining results from multiple MS/MS search methodologies._ J
Proteome Res. 2008 Jan;7(1):245-53. doi: 10.1021/pr070540w. PMID: 18173222.

**Searle BC**, Dasari S, Wilmarth PA, Turner M, Reddy AP, David LL, Nagalla SR.
_Identification of protein modifications using MS/MS de novo sequencing and the
OpenSea alignment algorithm._ J Proteome Res. 2005 Mar-Apr;4(2):546-54. doi:
10.1021/pr049781j. PMID: 15822933.

**Searle BC**, Dasari S, Turner M, Reddy AP, Choi D, Wilmarth PA, McCormack AL,
David LL, Nagalla SR. _High-throughput identification of proteins and
unanticipated sequence modifications using a mass-based alignment algorithm for
MS/MS de novo sequencing results._ Anal Chem. 2004 Apr 15;76(8):2220-30. doi:
10.1021/ac035258x. PMID: 15080731.
